packages/custom-webpack/src/extract-i18n/index.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
No need in this file (unless you tell me there is an explicit need)

Reply:
I doubled checked an you're correct, file removed.